src/cmd/build.rs

```diff
@@ -3,7 +3,7 @@ use std::env;
 use errors::Result;
 use site::Site;
 
-use console;
+use super::super::console;
```

Comment:
can you not use `crate::console`?

Reply:
I tried that initially but there were a number of similar errors at the time, so it *looked* like it didn't work (though it apparently did).  I've switched the relative use statements from `super::super::` to `crate::` and pushed a new commit.